packages/destination-actions/src/destinations/google-enhanced-conversions/functions.ts

```diff
@@ -69,6 +69,23 @@ export async function getCustomVariables(
   )
 }
 
+export async function getConversionActionId(
+  customerId: string | undefined,
+  auth: any,
+  request: RequestClient
+): Promise<ModifiedResponse<QueryResponse[]>> {
+  return await request(`https://googleads.googleapis.com/v14/customers/${customerId}/googleAds:searchStream`, {
```

Comment:
await is not needed if you are not processing the response.
```suggestion
  return request(`https://googleads.googleapis.com/v14/customers/${customerId}/googleAds:searchStream`, {
```